### PR TITLE
git-clone: add --recurse-submodules

### PR DESCRIPTION
**Pull request by mansoorbarri**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

page edited: `pages/common/git-clone.md`

### CODE REVIEW

File: pages/common/git-clone.md [hunk 1]
```diff
@@ -15,13 +15,9 @@
...
 
-- Clone a local repository:
+- Clone a local repository quietly:
 
-`git clone --local {{path/to/local/repository}}`
-
-- Clone quietly:
-
-`git clone --quiet {{remote_repository_location}}`
+`git clone --local --quiet {{path/to/local/repository}}`
```
Comment (vitorhcl) [inline pages/common/git-clone.md:20]: We only allow at most 8 examples per page, how about merging these 2 examples?

File: pages/common/git-clone.md [hunk 2]
```diff
@@ -34,3 +30,7 @@
 - Clone an existing repository using a specific SSH command:
 
 `git clone --config core.sshCommand="{{ssh -i path/to/private_ssh_key}}" {{remote_repository_location}}`
+
+- Automatically initialize and update each submodule in the repository:
```
Comment (vitorhcl) [inline pages/common/git-clone.md:34]:
```suggestion
- Automatically initialize and clone each submodule in the repository:
```
I think this is clearer

File: pages/common/git-clone.md (hunk 1 above)
Comment (mansoorbarri) [reply pages/common/git-clone.md:20]: yes makes sense

**Review by vitorhcl — APPROVED** [review]
LGTM! Let's see what the others think

**Comment by fazlearefin** [comment]
Maybe the description can be clarified a bit more?

Instead of

```
Automatically initialize and clone each submodule in the repository:
```

This one as if the submodules themselves contain more nested submodules, they will be cloned as wee

```
Automatically initialize and clone each submodule in the repository recursively:
```

**Review by fazlearefin — APPROVED** [review]

**Comment by mansoorbarri** [comment]
this is similar to `--recursive` as mentioned in #12244. closing pr.

**Pull request closed without merging**